schema.graphql

```diff
@@ -478,6 +478,33 @@ type ProjectMinterConfiguration @entity {
   maxInvocations: BigInt
 }
 
+type Bid @entity {
+  "Unique identifier made up of for SEA - {minter contract address}-{bidder}-{value}-{token} [TODO - do something different from RAM]"
+  id: ID!
+
+  "The associated project"
+  project: Project!
+
+  "The associated minter"
+  minter: Minter!
+
+  token: Token
+
+  "The associated account"
+  bidder: Account!
+
+  "The value of the bid"
+  value: BigInt!
```

Comment:
@yoshiwarab can you remind me if having a price here will create problems downstream? I can't remember if we only needed a price-as-string in the json fields, or if we should also be using price here.

Reply:
Yeah this should be fine and will just come through as a string when queried. As long as the corresponding DB field is numeric (supports very large numbers) we should be good.